src/components/enterprise-page/EnterprisePage.jsx

```diff
@@ -17,6 +18,10 @@ const EnterprisePage = ({ children }) => {
   useEffect(() => {
     if (isDefinedAndNotNull(enterpriseCustomer)) {
       pushUserCustomerAttributes(enterpriseCustomer);
+
+      // Set custom attributes for logging service
+      const loggingService = getLoggingService();
+      loggingService.setCustomAttribute('enterprise_customer_uuid', enterpriseCustomer.uuid);
```

Comment:
[inform] Ensures `enterprise_customer_uuid` is tagged for Datadog RUM events (e.g., to facet/query session replays by specific enterprise customers).